src/plan_bot.py

```diff
@@ -76,25 +76,43 @@ def process_post(
             plan_confidence = match_info["confidence"]
             plan = match_info["plan"]
             plan_id = plan["id"]
-
-            # If plan is matched with confidence, build and send reply
-            if match:
+            post_parent_id = False # Could be "" unless constructing URLs with it later
```

Comment:
Nit: I think typical python would be `None`

Reply:
You're right, it is None, but I wasn't sure what the result would be if we end up pushing to Firestore, and then pulling it back again. Would Firestore interpret it as a strong, and then send back "None" later, which would then be interpreted by python as a string ("None") instead of Python's "null", which is None? If we used an empty string, we could still concat an empty string into a URL (or whatever we needed it for). But, if we got "None" back from Firestore instead of None then that has potential for hatching a bug later. By using False, we can check if post_parent_id safely across the board. At least that was my logic for why I did it that way.

Reply:
I'm 99% sure that FireStore will store it as `null` and send back `None`!

Reply:
If I understand FireStore doesn't really have any concept of a field being a certain type, just that each value has a type, and so there's no reason for it to convert to string

https://firebase.google.com/docs/firestore/manage-data/data-types